ops/model.py

```diff
@@ -938,7 +971,7 @@ def _is_mutable(self):
         return False
 
     def __setitem__(self, key: str, value: str):
-        if not self._is_mutable():
+        if not self._is_writable():
             raise RelationDataError(
                 'cannot set relation data for {}'.format(self._entity.name))
```

Comment:
Feels like this error message text should be consistent with the get/read one below.